### PR TITLE
Style2

### PR DESCRIPTION
**Pull request by ilyafefelov**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -13,13 +15,25 @@
...
     "build-sass": "node-sass-chokidar src/ -o src/",
+    "stylelint-fix-sass": "stylelint --fix 'src/**/*.scss'",
+    "stylelint-dev": "stylelint 'src/**/*.scss' --cache --cache-location './.stylelintcache'; exit 0",
+    "stylelint-build": "stylelint 'src/**/*.scss'",
+    "watch-styles-sass": "npm-watch",
     "start-js": "react-scripts start",
-    "start": "npm-run-all -p watch-compile-sass start-js",
-    "build": "npm run build-sass && react-scripts build",
+    "start": "npm-run-all -p watch-styles-sass watch-compile-sass start-js",
+    "build": "npm run stylelint && npm run build-sass && react-scripts build",
```
Comment (zozich) [inline client/package.json]: You will encounter same error here. You don't have script called `stylelint`, you have `stylelint-build`

Comment (ilyafefelov) [reply client/package.json]: updated

**Comment by ilyafefelov** [comment]
[Configure Stylelint on front-end](https://trello.com/c/iYeDFOmQ/10-configure-stylelint-on-front-end)

**Comment by ilyafefelov** [comment]
updates package.json with stylelint-build

**Pull request merged**